modules/claims_api/spec/requests/v1/rswag_intent_to_file_request_spec.rb

```diff
@@ -0,0 +1,319 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+require 'rails_helper'
+require_relative '../../support/swagger_shared_components'
+
+describe 'Intent to file' do # rubocop:disable RSpec/DescribeClass
+  path '/forms/0966' do
+    post 'Submit form 0966 Intent to File.' do
+      tags 'Intent to File'
+      operationId 'post0966itf'
+      security [bearer_token: []]
+      consumes 'application/json'
+      produces 'application/json'
+      post_description = <<~VERBIAGE
+        Establishes an intent to file for disability compensation, burial, or pension claims.
+      VERBIAGE
+      description post_description
+
+      parameter SwaggerSharedComponents.header_params[:veteran_ssn_header]
+      let(:'X-VA-SSN') { '796-04-3735' }
+
+      parameter SwaggerSharedComponents.header_params[:veteran_first_name_header]
+      let(:'X-VA-First-Name') { 'WESLEY' }
+
+      parameter SwaggerSharedComponents.header_params[:veteran_last_name_header]
+      let(:'X-VA-Last-Name') { 'FORD' }
+
+      parameter SwaggerSharedComponents.header_params[:veteran_birth_date_header]
+      let(:'X-VA-Birth-Date') { '1986-05-06T00:00:00+00:00' }
+      let(:Authorization) { 'Bearer token' }
+
+      parameter name: :data, in: :body, required: true, schema: {
+        type: :object,
+        properties: {
+          data: {
+            type: :hash,
+            description: 'JSON API Payload of Veteran being submitted',
+            example: 'DO WE WANT AN EXAMPLE HERE???'
```

Comment:
yeah, mostly since our current doc has one, and the example is used to provide an example curl call

Reply:
weirdly, the schema for this endpoint request looks strange, like its looking at the example to build off of or something? that doesnt really make sense but the request schema does look off for whatever reason

Reply:
This should be fixed now.